### ISSUE TITLE
failure on initial run

### ISSUE THREAD

**Issue by tkellen** [programble/careen#2]
Using the example provided in the repo, the first run fails:

```
Unhandled rejection TypeError: Cannot read property 'timestamp' of undefined
    at /Users/tkellen/careen/lib/command/apply.js:49:65
    at _filter (/Users/tkellen/careen/node_modules/ramda/dist/ramda.js:338:17)
    at /Users/tkellen/careen/node_modules/ramda/dist/ramda.js:3934:23
    at Object.f2 [as filter] (/Users/tkellen/careen/node_modules/ramda/dist/ramda.js:238:24)
    at /Users/tkellen/careen/lib/command/apply.js:49:18
    at eval (eval at <anonymous> (/Users/tkellen/careen/node_modules/bluebird/js/main/join.js:23:16), <anonymous>:6:20)
    at tryCatcher (/Users/tkellen/careen/node_modules/bluebird/js/main/util.js:26:23)
    at Holder.checkFulfillment (/Users/tkellen/careen/node_modules/bluebird/js/main/join.js:51:40)
    at Promise.eval (eval at <anonymous> (/Users/tkellen/careen/node_modules/bluebird/js/main/join.js:13:16), <anonymous>:6:20)
    at Promise._settlePromiseAt (/Users/tkellen/careen/node_modules/bluebird/js/main/promise.js:575:21)
    at Promise._settlePromises (/Users/tkellen/careen/node_modules/bluebird/js/main/promise.js:693:14)
    at Async._drainQueue (/Users/tkellen/careen/node_modules/bluebird/js/main/async.js:123:16)
    at Async._drainQueues (/Users/tkellen/careen/node_modules/bluebird/js/main/async.js:133:10)
    at Immediate.Async.drainQueues [as _onImmediate] (/Users/tkellen/careen/node_modules/bluebird/js/main/async.js:15:14)
    at processImmediate [as _immediateCallback] (timers.js:358:17)
```

...all subsequent runs (including a complete rollback and application) work fine after this, though!

**Comment by causal-agent** [comment]
Fixed in v0.1.1.

**Comment by tkellen** [comment]
thanks!
